filer/static/filer/js/base.js

```diff
@@ -88,12 +88,16 @@ Cl.mediator = new Mediator();
                 }
             }, 100);
 
-            actionSelect.on('click', function () {
-                if (!navigatorTable.hasClass('selected')) {
-                    actionList.addClass('action-selected');
-                } else {
-                    actionList.removeClass('action-selected');
-                }
+            actionSelect.on('change', function () {
+                // setTimeout makes sure that change event fires before click event which is reliable to admin
```

Comment:
setTimeout makes sure that change event fires *after* click event which is required for admin scripts to reliably finish manipulations with rows